docs/user/aws/install_upi.md

```diff
@@ -253,9 +268,47 @@ The CSR can be approved by using
 oc adm certificate approve <csr_name>
 ```
 
-## Configure Router for UPI DNS
+## Configure Router for UPI
+
+The Ingress operator manages DNS and LoadBalancers. It makes use of tags on HostedZones to identify which public and 
+private zones are to be updated from the cluster by the operator as objects are created in the cluster. 
+
+The tags used for finding HostedZones used by the operator
+are fulfilled by the CloudFormation template [here](../../../upi/aws/cloudformation/02_cluster_infra.yaml).
+
+An example of the `spec` for DNS configuration is below:
+
+```
+$ oc get dns -o yaml
+apiVersion: v1
+items:
+- apiVersion: config.openshift.io/v1
+  kind: DNS
+  metadata:
+    creationTimestamp: 2019-03-28T12:31:10Z
+    generation: 1
+    name: cluster
+    namespace: ""
+    resourceVersion: "395"
+    selfLink: /apis/config.openshift.io/v1/dnses/cluster
+    uid: 5e51dd25-5155-11e9-befc-02d75ce1a902
+  spec:
+    baseDomain: test.example.com
+    privateZone:
+      tags:
+        Name: test-r69hh-int
+        kubernetes.io/cluster/test-r69hh: owned
```

Comment:
nit: of this whole YAML manifest, this is the only line we want, right?  It would be nice to use JSONPath to limit the output to just that, because it would go stale more slowly.  But again, not worth holding up the PR.